modules/doc/content/getting_started/installation/manual_petsc.md

```diff
@@ -1,64 +1,71 @@
-## PETSc
+## Compile PETSc
 
-Download PETSc [!package!petsc_default]
+By default, we use PETSc submodule inside of MOOSE as our nonlinear/linear solvers.
+PETSc can be built using the following script:
 
-!package code
-cd $STACK_SRC
-curl -L -O http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-__PETSC_DEFAULT__.tar.gz
-tar -xf petsc-__PETSC_DEFAULT__.tar.gz -C .
+```bash
+cd ~/projects/moose
 
+unset PETSC_DIR PETSC_ARCH
```

Comment:
Why do we not unset these in the script?

Would be nice to see if they're set, unset them, and add a warning that they are being unset.

Reply:
Oh, we allow users to set PETSC_DIR and PETSC_ARCH. There are many use cases we need to be able to set PETSC_ARCH.  

The instructions here are for regular users (inexperienced users ).